pkg/internal/apis/config/default.go

```diff
@@ -67,6 +67,10 @@ func SetDefaultsCluster(obj *Cluster) {
 			obj.Networking.ServiceSubnet = "fd00:10:96::/112"
 		}
 	}
+	// default the KubeProxyMode using iptables as it's already the default
+	if obj.Networking.KubeProxyMode == "" {
+		obj.Networking.KubeProxyMode = "iptables"
```

Comment:
we should use an enum, see the IPFamily for reference https://github.com/kubernetes-sigs/kind/pull/1595/files#diff-4686ef90e1f4b2babdb7c7bf7997b62dL136-L143

Reply:
I've made some adjustments here, let me know if something else is needed :)